src/views/Community/components/MemberCard/index.tsx

```diff
@@ -15,26 +14,21 @@ interface OwnProps {
 
 const MemberCard = ({ user, match }: OwnProps) => {
   return (
-    // <Link key={user._id} className={styles.memberLink} to={`/member/${user._id}`}>
-    <Card className={styles.memberCard}>
-      <Space direction="vertical" align="center">
-        <Gravatar
-          className={styles.userGravatar}
-          circle
-          placeholder={DEFAULT_GRAVATAR_PLACEHOLDER}
-          email={user.email || ''}
-        />
-        <Typography.Title className={styles.memberCardName} level={5}>
-          <Highlighter textToHighlight={formatName(user) ?? ''} searchWords={[match]} />
-        </Typography.Title>
-        {user.institution && (
-          <Typography.Text type="secondary" className={styles.memberAffiliation}>
-            <Highlighter textToHighlight={user.institution} searchWords={[match]} />
-          </Typography.Text>
-        )}
-      </Space>
-    </Card>
-    // </Link>
+    <Link key={user._id} className={styles.memberLink} to={`/member/${user._id}`}>
+      <Card className={styles.memberCard}>
+        <Space direction="vertical" align="center">
+          <Gravatar className={styles.userGravatar} circle email={user.email || ''} />
+          <Typography.Title className={styles.memberCardName} level={5}>
+            <Highlighter textToHighlight={formatName(user) ?? ''} searchWords={[match]} />
```

Comment:
maybe `formatName` could handle the fallback value `''`

Reply:
`  user.firstName && user.lastName ? `${user.firstName} ${user.lastName}` : user.email;` seems like it need to return the email by default, so maybe review it later when removing persona